scripts/build_windows.cmd

```diff
@@ -16,11 +16,13 @@ where choco
 if %errorlevel% neq 0 (echo Error: choco not found && goto :error)
 
 choco install InnoSetup
+choco install git
 call refreshenv
 where iscc
 if %errorlevel% neq 0 (echo Error: Couldn't find Inno Setup compiler. && goto :error)
 
 echo ====== Installing requirements... ======
+call git patch scripts/patches/windows.patch
```

Comment:
https://github.com/iterative/dvc/pull/2773#discussion_r344504643 Same here, we could totally
```
echo 'PKG = "exe"' >dvc\build.py
```
:slightly_smiling_face: